website/src/pages/news.tsx

```diff
@@ -22,233 +22,65 @@ export default class NewsPage extends React.Component<any, any> {
             'Sourcegraph in the news.'
```

Comment:
You don't seem to be using this `const` anywhere.

Reply:
fixted